numpy/core/_ufunc_config.pyi

```diff
@@ -0,0 +1,43 @@
+import sys
+from typing import Optional, Union, Callable, Any
+
+if sys.version_info >= (3, 8):
+    from typing import Literal, Protocol, TypedDict
+else:
+    from typing_extensions import Literal, Protocol, TypedDict
+
+_ErrKind = Literal["ignore", "warn", "raise", "call", "print", "log"]
+_ErrFunc = Callable[[str, int], Any]
```

Comment:
The signatures of `_ErrFunc` and `_SupportsWrite` could be made a bit stricter, mandating `None` as the returned value.
Any thoughts?

Reply:
Seems reasonable, but I'll leave that decision to you.